### DISCUSSION TITLE
Math Solver 数学求解器

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#422] [Show and tell]
@langningchen @fossabot @boomzero @zhouyiqing0304 @PythonSmall-Q
~继续 `Show and tell` 😊~

https://github.com/langningchen/Math 这个仓库，用的Microsoft Math Solver的接口， https://math.chenlangning.me 演示，点个 Star 谢谢

**Comment by boomzero** [comment]
Emm, I saw it yesterday

**Comment by langningchen** [comment]
@boomzero 我看到你点了 Star ，后来有取消了

**Comment by boomzero** [comment]
点错了(

**Comment by langningchen** [comment]
@boomzero 都已经8 Stars了，你要不也来一个？

**Reply by boomzero** [reply]
done

**Reply by langningchen** [reply]
@boomzero Thanks

**Comment by PythonSmall-Q** [comment]
Stared

**Reply by langningchen** [reply]
@PythonSmall-Q Thanks
